src/main/resources/application-worker.yaml

```
@@ -24,3 +24,9 @@ rhsm-subscriptions:
     keystore: file:${SUBSCRIPTION_KEYSTORE:}
     keystorePassword: changeit
     maxConnections: ${SUBSCRIPTION_MAX_CONNECTIONS:100}
+  product:
+    useStub: ${STUB_PRODUCT:true}
+    url: http://${PRODUCT_HOST:localhost}:${PRODUCT_PORT:8080}/svcrest/product/v3
+    keystore: file:${PRODUCT_KEYSTORE:}
+    keystorePassword: changeit
```

Can you please expose the keystore password as an environment variable?
```suggestion
    keystorePassword: ${PRODUCT_KEYSTORE_PASSWORD:changeit}
```

---

Done. I also applied this change to the subscription config above.